setup.py

```diff
@@ -55,7 +55,7 @@ def main():
         py_modules=['venv_update', 'pip_faster'],
         packages=find_packages(exclude=('tests*',)),
         install_requires=[
-            'pip>=10.0.0',
+            'pip>=10.0.0,<=18.0',
```

Comment:
iirc this also needs to be specified in `import_tests.txt` if it isn't there already (at least, the `sed` command hints at it -- if it's not needed there, adjust the `sed` command too!)

Reply:
Good catch ... completely forgot about it ;(